README.md

```diff
@@ -120,25 +119,100 @@ fastify.register(jwt, {
   }
 })
 // secret as an object of RSA keys (without passphrase)
-// assuming the key files are inside a certs directory and loaded as strings
+// the files are loaded as strings
 fastify.register(jwt, {
   secret: {
-    private: readFileSync(`${path.join(__dirname, 'certs')}/private.key`, 'utf8')
-    public: readFileSync(`${path.join(__dirname, 'certs')}/public.key`, 'utf8')
+    private: readFileSync('path/to/private/cert.key', 'utf8'),
+    public: readFileSync('path/to/public/cert.key', 'utf8')
```

Comment:
I prefer actual copy-paste examples, can you restore path?

Reply:
done. ^^